### ISSUE TITLE
[Question]: 2 or more arguments

### ISSUE THREAD

**Issue by ek-ex** [danielmiessler/Fabric#1177]
### What is your question?

Is there a way to create templates that would take more than 1 argument? How can I achieve this?

**Comment by mattjoyce** [comment]
Yes, you can have multiple variables in either the input or pattern.

Variables should be uniquely named.

Here's an example.

> Echo " {{First}} {{Second}}" | ./fabric -v=First:foo -v=Second:bah
